Sources/AlgoliaSearch-Client/Index.swift

```diff
@@ -535,10 +535,10 @@ import Foundation
     @objc(browseFromCursor:requestOptions:completionHandler:)
     @discardableResult public func browse(from cursor: String, requestOptions: RequestOptions? = nil, completionHandler: @escaping CompletionHandler) -> Operation {
         let path = "1/indexes/\(urlEncodedName)/browse"
-        let urlParameters = ["cursor": cursor]
-        return client.performHTTPQuery(path: path, urlParameters: urlParameters, method: .GET, body: nil, hostnames: client.readHosts, requestOptions: requestOptions, completionHandler: completionHandler)
+        let bodyParams = ["params": "cursor=\(cursor)"]
```

Comment:
You don't have to nest it under `params`, you can do:
```swift
let bodyParams = ["cursor": cursor]
```

Both will work so it's not so important, but we're trying to remove usage of `params: allTheThingsEncoded` to make the logs easier to read.